pkg/cmd/adm/unregister_member_test.go

```diff
@@ -21,7 +21,7 @@ func TestUnregisterMemberWhenAnswerIsY(t *testing.T) {
 	ctx := clicontext.NewCommandContext(term, newClient)
 
 	// when
-	err := UnregisterMemberCluster(ctx, "member1", func(ctx *clicontext.CommandContext, clusterName string) error {
+	err := UnregisterMemberCluster(ctx, "member1", func(ctx *clicontext.CommandContext, clusterName string, cfcGetter ConfigFlagsAndClientGetterFunc) error {
```

Comment:
minor - but I would set those params to `_` if not used
```suggestion
	err := UnregisterMemberCluster(ctx, "member1", func(_ *clicontext.CommandContext, _ string, _ ConfigFlagsAndClientGetterFunc) error {
```

Reply:
and in the other places as well

Reply:
updated it to all places , see [a05065e](https://github.com/kubesaw/ksctl/pull/90/commits/a05065ec42b65641228054641e05b47a2fe10450)